deploy/helm/flightctl/templates/flightctl-api-config.yaml

```diff
@@ -17,42 +17,25 @@ data:
         address: :3443
         agentEndpointAddress: :7443
         agentGrpcAddress: :7444
-        {{- if .Values.api.hostNames.api }}
-        baseUrl: https://{{ .Values.api.hostNames.api }}{{ if .Values.api.nodePorts.api }}:{{ .Values.api.nodePorts.api }}{{ end }}/
+        {{- if eq .Values.global.exposeServicesMethod "nodePort" }}
+        baseUrl: https://{{ include "flightctl.getBaseDomain" . }}:{{ .Values.global.nodePorts.api }}/
+        baseAgentEndpointUrl: https://{{ include "flightctl.getBaseDomain" . }}:{{ .Values.global.nodePorts.agent }}/
+        baseAgentGrpcUrl: https://{{ include "flightctl.getBaseDomain" . }}:{{ .Values.global.nodePorts.grpc }}/
+        baseUIUrl: {{ include "flightctl.getUIHttpScheme" . }}://{{ include "flightctl.getBaseDomain" . }}:{{ .Values.global.nodePorts.ui }}/
         {{- else }}
-        baseUrl: https://api.{{ include "flightctl.getBaseDomain" . }}{{ if .Values.api.nodePorts.api }}:{{ .Values.api.nodePorts.api }}{{ end }}/
-        {{- end }}
-        {{- if .Values.api.hostNames.agent }}
-        baseAgentEndpointUrl:  https://{{ .Values.api.hostNames.agent }}{{ if .Values.api.nodePorts.agent }}:{{ .Values.api.nodePorts.agent }}{{ end }}/
-        {{- else }}
-        baseAgentEndpointUrl:  https://agent-api.{{ include "flightctl.getBaseDomain" . }}{{ if .Values.api.nodePorts.agent }}:{{ .Values.api.nodePorts.agent }}{{ end }}/
-        {{- end }}
-        {{- if .Values.api.agentGrpcBaseURL }}
-        baseAgentGrpcUrl:  {{ .Values.api.agentGrpcBaseURL }}
-        {{- else }}
-        baseAgentGrpcUrl:  grpcs://agent-grpc.{{ include "flightctl.getBaseDomain" . }}{{ if .Values.api.nodePorts.grpc }}:{{ .Values.api.nodePorts.grpc }}{{ end }}
-        {{- end }}
-        {{- if .Values.api.baseUIUrl }}
-        baseUIUrl:  {{ .Values.api.baseUIUrl }}
-        {{- else if eq .Values.global.target "acm" }}
-        baseUIUrl:  https://console-openshift-console.{{ include "flightctl.getBaseDomain" (merge . (dict "noNs" "true")) }}/edge
+        baseUrl: https://api.{{ include "flightctl.getBaseDomain" . }}/
+        baseAgentEndpointUrl: https://agent-api.{{ include "flightctl.getBaseDomain" . }}/
+        baseAgentGrpcUrl: https://agent-grpc.{{ include "flightctl.getBaseDomain" . }}/
```

Comment:
```suggestion
        baseAgentGrpcUrl: grpcs://agent-grpc.{{ include "flightctl.getBaseDomain" . }}/
```

Reply:
We could btw fix the agent to support "https" in the scheme, but I think we only parse for grpcs... it's the same in the end... :-/

Reply:
fixed